src/asm/lexer.c

```diff
@@ -1471,15 +1487,13 @@ static void readString(void)
 
 		case '{': /* Symbol interpolation */
 			shiftChars(1);
+			lexerState->disableMacroArgs = false;
 			char const *ptr = readInterpolation();
 
-			if (ptr) {
-				while (*ptr) {
-					if (i == sizeof(yylval.tzString))
-						break;
+			if (ptr)
+				while (*ptr && i < sizeof(yylval.tzString))
```

Comment:
This change was necessary to make checkpatch.pl stop warning "Too many leading tabs - consider code refactoring".

Reply:
You *did* consider refactoring. :)